groups/instance/security_groups.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_security_group" "instance_security_group" {
+  name        = "${var.environment}-${var.service}-instance"
+  description = "Allow TLS inbound traffic"
+  vpc_id      = data.aws_vpc.placement.id
+
+  ingress {
+    description      = "Artifactory Ingress from permitted CIDRs"
```

Comment:
We can simplify this to something like `Artifactory`. The rest is implicit as we're defining an ingress rule